src/js/angular/clustermanagement/templates/modal/remove-nodes-dialog.html

```diff
@@ -0,0 +1,44 @@
+<!-- Start remove nodes from cluster modal -->
+
+<div class="modal-header">
+    <button type="button" class="close" ng-click="cancel()"></button>
+    <h3 class="modal-title">{{'cluster_management.remove_nodes_dialog.title' | translate}}</h3>
+</div>
+<form ng-submit="ok()" name="form">
+    <div class="modal-body">
+        <h5>{{'cluster_management.remove_nodes_dialog.content' | translate}}</h5>
+        <div class="row">
+            <div class="col-xs-12 mb-1 mt-1">
+                <h5>{{'cluster_management.cluster_page.cluster_nodes_list' | translate}}</h5>
+                <ul ng-if="nodes.length" class="list-group nodes-list">
+                    <li ng-repeat="node in nodes" ng-class="{'list-group-item-danger': nodesToRemove.includes(node)}"
```

Comment:
I generally somehow don't like such approach to resolve the marker css class with an array function call in a repeater. If we have a more rich model of these node lists, we would be able to precalculate such things and just bind to a model property instead of resolving it runtime.

Reply:
I refactored the controller to make the nodes an object with a `shouldRemove` property